src/link/assign.c

```diff
@@ -42,7 +67,7 @@ area_Avail(SLONG bank)
 	return (r);
 }
 
-SLONG 
+bool
```

Comment:
Just to clarify, if you'd prefer me not to use `bool` here, would you rather I revert it back to how it was (i.e. returning either `org` or `-1`)?